synctl

```diff
@@ -228,6 +238,7 @@ def main():
     start_stop_synapse = True
 
     if cache_factor:
+        logger.warning(SYNCTL_CACHE_FACTOR_WARNING)
```

Comment:
I don't believe `logger` is necessary here - `write(SYNCTL_CACHE_FACTOR_WARNING)` should do the same thing.

Reply:
@anoadragon453 Are you sure? I used the logger because it is also used for this warning:

https://github.com/matrix-org/synapse/blob/0cc3bf97b4399234cf20f52ae4c09d03661225ff/synapse/config/key.py#L74-L78

Reply:
`synctl` is a separate script from Synapse. It's used to start up Synapse and run it as a daemon. We do use `logging` extensively in Synapse, but not in `synctl` (currently).

I don't know if there's an advantage to using logging over `write()` commands in `synctl`, but if there is we should change them all out - and in a separate PR.

Reply:
:+1: